styles/command-palette.less

```diff
@@ -0,0 +1,7 @@
+@import "ui-variables";
+
+// Highlight matched text
+.select-list .list-group .character-match {
```

Comment:
So yeah, this should probably just be `.command-palette .list-group .character-match {`